cms/grading/ScoreType.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, parameters, public_testcases):
         self.max_score, self.max_public_score, self.ranking_headers = \
             self.max_scores()
 
-    def get_html_details(self, score_details, translator=None):
+    def get_html_details(self, score_details, translator=None, AWS=False, interface_type=None):
```

Comment:
Do you even need AWS? The only thing that passes an interface_type will be CWS, so you can probably just rely on that.